src/cfclient_evaluator.erl

```diff
@@ -172,10 +172,10 @@ number_variation(FlagIdentifier, Target, DefaultValue) ->
   end.
 
 
-%% TODO - not implemented
--spec json_variation(Identifier :: binary(), Target :: target(), DefaultValue :: binary()) -> binary().
+-spec json_variation(Identifier :: binary(), Target :: target(), DefaultValue :: map()) -> map().
 json_variation(FlagIdentifier, Target, DefaultValue) ->
-  not_implemented.
+  Variation = evaluate_flag(FlagIdentifier, Target),
+  jsx:decode(Variation, []).
```

Comment:
May need to catch a `badarg` exception here as part of the error handling improvement work.

Reply:
Noted @bmjen